adafruit_lis3dh.py

```diff
@@ -93,7 +101,9 @@ class LIS3DH:
       the LIS3DH INT2 interrupt pin
     """
 
-    def __init__(self, int1=None, int2=None):
+    def __init__(
+        self, int1: digitalio.DigitalInOut = None, int2: digitalio.DigitalInOut = None
```

Comment:
These should be wrapped with `Optional` since they can take and default to `None`.

Reply:
Tks see https://github.com/adafruit/Adafruit_CircuitPython_LIS3DH/pull/76/commits/59c669238f330e5703bb89641199c02303db38c7